### ISSUE TITLE
Suggestion: Make explicit the version of Erlang in the book

### ISSUE THREAD

**Issue by hirotnk** [happi/theBeamBook#77]
As stated [here](https://github.com/happi/theBeamBook#linking-to-otperlang-source-code), it may be helpful to explicitly state somewhere in the book which version it targets.
Then readers can guess about the contents of the book even after the implementation is changed later. Since sometimes there are major changes that affect the contents of this book.

**Comment by happi** [comment]
Yes, I have been thinking about this.
One problem is that right now there are old pieces of text written when Erlang was on version 14 and and 15 and newer parts that cover version 18 and 19.
Perhaps we would need to tag each chapter with which Erlang version it is updated to?

The goal right now is to cover Erlang 19 but we can not really state in the preface or introduction that the book covers 19 since some chapters have not been revised to that version.

I am all for making this information clear in some way so any suggestion on how to do it is welcome.

**Comment by garazdawi** [comment]
I think a [NOTE] at the start of each chapter describing which version it was written for would be great.

**Comment by KronicDeth** [comment]
OTP 21 changes how messages are added to other processes, but I'm not 💯 if the diagrams need to change.

**Comment by hirotnk** [comment]
Agreed with @garazdawi 's comment above, I think just adding OTP version to [NOTE] would work. If there is more to say, we can always add descriptions to it.

**Comment by happi** [comment]
How bad did it look in the PDF? (or did it not even build?)

**Comment by hirotnk** [comment]
It built, but looks as below:
<img width="602" alt="Screen Shot 2020-03-11 at 20 27 12" src="https://user-images.githubusercontent.com/366615/76484427-00289700-63d7-11ea-8761-71778d35edfb.png">

- I see it's using dblatex, but asciidoctor-pdf can not be used?
- Also, shaape does not look being maintained anymore (https://github.com/christiangoltz/shaape/issues/20), the last commit was 2014. Does it make sense to look for an alternative ?

**Comment by happi** [comment]
Yes it might be a good time to update the build process. My plan is to get to finish the missing chapters in the near future and I will work more actively on the book now. 

If you have some suggestions please let me know.

**Comment by hirotnk** [comment]
The addition of missing chapters would be great. Does it help in any way to try to improve the formatting or layout at the moment, or better to wait for now?

**Comment by happi** [comment]
If you want to do it please go ahead, I would appreciate it.

**Comment by hirotnk** [comment]
Got it.

**Comment by hirotnk** [comment]
As part of https://github.com/happi/theBeamBook/pull/135, Admonition icon works in pdf as below:
<img width="623" alt="Screen Shot 2020-04-18 at 3 02 23 PM" src="https://user-images.githubusercontent.com/366615/79672353-e9b7ed00-8185-11ea-9f62-a330c47f32c2.png">

On HTML, it looks as below:
<img width="999" alt="Screen Shot 2020-04-18 at 3 05 39 PM" src="https://user-images.githubusercontent.com/366615/79672374-25eb4d80-8186-11ea-96d3-3c58f8e61ac5.png">
